README.md

```diff
@@ -15,9 +15,27 @@ from scratch as well as keep it up-to-date over time.
 
 ### Prerequisites
 
+* `brew`
+
+```bash
+xcode-select -–install
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+>see: [Install Homebrew without XCODE](https://www.codeandpeace.com/install-homebrew-without-xcode/)
+
 * `git` must be installed
+
+```bash
```

Comment:
yeah, windows for sure. couldnt remember on the other two. no chocolatey instructions? ;)